corehq/apps/domain/views.py

```diff
@@ -1269,9 +1269,10 @@ def parent_pages(self):
 
     @property
     def steps(self):
+        edition_name = u" (%s)" % self.edition_name if self.edition_name else ""
         return [
             {
-                'title': _("1. Select a Plan%s") % (" (%s)" % self.edition_name if self.edition_name else ""),
+                'title': _(u"1. Select a Plan{0}".format(edition_name)),
```

Comment:
I think translating a dynamically generated string like this will not work, hence the previous usage of % for string formatting after string formatting.